client/packages/electron/src/electron.ts

```diff
@@ -0,0 +1,100 @@
+import { app, BrowserWindow, ipcMain } from 'electron';
+import startDiscovery from 'multicast-dns';
+import { IPC_MESSAGES } from './shared';
+import ip from 'ip';
+import { FrontEndHost, frontEndHostUrl } from '@openmsupply-client/common/src/hooks/useElectronClient';
+
+const SERVICE_NAME = '_omsupply._tcp.local';
+const SERVICE_TYPE = 'TXT';
+const QUERY_INTERVAL = 1000;
+
+const discovery = startDiscovery();
+
+let connectedServer: FrontEndHost | null = null;
+let isDiscovering = false;
+
+// This allows TypeScript to pick up the magic constant that's auto-generated by Forge's Webpack
+// plugin that tells the Electron app where to look for the Webpack-bundled app code (depending on
+// whether you're running in development or production).
+declare const MAIN_WINDOW_WEBPACK_ENTRY: string;
+declare const MAIN_WINDOW_PRELOAD_WEBPACK_ENTRY: string;
+
+// Handle creating/removing shortcuts on Windows when installing/uninstalling.
+if (require('electron-squirrel-startup')) {
+  // eslint-disable-line global-require
+  app.quit();
+}
+
+const start = (): void => {
+  // Create the browser window.
+  const window = new BrowserWindow({
+    height: 600,
```

Comment:
the size is fairly conservative.. I guess people can resize though 🤷

Reply:
I guess, this is default, btw, as in i didn't put any though into it